frontend/src/metabase/admin/people/containers/UserSuccessModal.jsx

```diff
@@ -22,6 +23,10 @@ class UserSuccessModal extends React.Component {
   }
   render() {
     const { onClose, user, temporaryPassword } = this.props;
+    const isEeSsoConfigured =
+      MetabaseSettings.isEnterprise() &&
+      MetabaseSettings.ssoConfigured() &&
+      !MetabaseSettings.passwordLoginEnabled();
```

Comment:
@ranquild Move everything so `settings.ts` and use similar logic in the at BE.
https://github.com/metabase/metabase/blob/2a502ae3a6f850b0f9dacefed0e1c8d0aad79687/src/metabase/models/user.clj#L247-L249